Configs/.config/hypr/windowrules.conf

```diff
@@ -4,6 +4,7 @@
 
 # See https://wiki.hyprland.org/Configuring/Window-Rules/
 
+windowrulev2 = opacity 0.90 0.90,class:^(.*)$
```

Comment:
this will affect the opacity for all application, and also multiply the opacity for the windows that are set below... so I don't think this should be default as it will interfere with readability... so better to leave it upto users to add the window rules.